src/platform/forms/save-in-progress/SaveInProgressErrorPage.jsx

```diff
@@ -19,6 +19,15 @@ import { toggleLoginModal } from '../../site-wide/user-nav/actions';
 // For now, this only handles loading errors, but it could feasibly be reworked
 //  to handle save errors as well if we need it to.
 class SaveInProgressErrorPage extends React.Component {
+  componentDidMount() {
+    if (
+      this.props.loadedStatus === LOAD_STATUSES.notAttempted &&
+      document.referrer.includes('/auth/login/callback/')
```

Comment:
We could probably leave the referrer check out. If the status is a value that we're not displaying any content for, then we want to send the user somewhere else no matter where they came from.